src/app/companies/companies.component.ts

```diff
@@ -1,10 +1,23 @@
-import { Component } from '@angular/core';
+import { COMPANIES_LIST } from './companies-list';
+import { Component, OnInit } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import { MatDialog } from '@angular/material';
+
+import { CompanyService } from './company.service';
+import { Company } from './companies.model';
 
 @Component({
   selector: 'app-companies',
   templateUrl: './companies.component.html',
   styleUrls: ['./companies.component.scss']
 })
 export class CompaniesComponent {
-  constructor() {}
+  companies: Observable<Company[]>;
+  constructor(private companyService: CompanyService) {
+    this.companies = this.companyService.getCompanies();
+  }
+
+  openWindow(url: string) {
```

Comment:
method isn't needed anymore, I think,

Reply:
@JWO719 really appreciate you taking the time to review PRs!

Reply:
thank you. It's a pleasure